packages/gatsby-source-wordpress/src/gatsby-browser.ts

```diff
@@ -34,6 +33,23 @@ export function onRouteUpdate(): void {
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let ReactDOM: any
```

Comment:
I typed this as any because it's either ReactDOM from React 17 or 18 (we don't know) but also we had ts-ignore comments on everything below anyway. Might as well any it and remove those comments